docs/open/QUICKSTART.md

```diff
@@ -1,63 +1,89 @@
 # Quick Start Guide
 
-If you’re new to Terraform and/or want to deploy DC/OS on AWS quickly and effortlessly - please follow this guide.  We’ll walk you through step-by-step on:
+If you’re new to Terraform and/or want to deploy DC/OS on AWS quickly and effortlessly - please follow this guide.  We’ll walk you through step-by-step on how to:
 
 
-- Creating a DC/OS OSS Cluster
-- Scaling the cluster
-- Upgrading the cluster
-- Deleting the cluster
+1) Create an Open Source DC/OS Cluster on AWS
+2) Scale the cluster to a larger number of nodes
+3) Upgrade the cluster to a newer version of DC/OS
+4) Destroy the cluster and all AWS resources associated with it
 
 # Prerequisites: 
-Terraform, cloud credentials, and SSH keys:
+Terraform, AWS cloud credentials, SSH keys
 
-## You’ll need Terraform.
-If you're on a Mac environment with homebrew installed, run this command.
+## Installing Terraform.
+If you're on a Mac environment with homebrew installed, simply run the following command:
 ```bash
 brew install terraform
 ```
 
-For help installing Terraform on a different OS, see [here](https://www.terraform.io/downloads.html):
+Once this command completes, you should be able to run the following command and see output consistent with the version of Terraform you have installed:
+```bash
+$ terraform version
+Terraform v0.11.8
+```
+
+For help installing Terraform on a different OS, please see [here](https://www.terraform.io/downloads.html):
 
+## Ensure you have your AWS Cloud Credentials Properly Set up
+I'm not sure exactly waht the best way to do this is. 
```

Comment:
Please take note of this. I can make the change, I just need to know what to put here.

Reply:
ideally a link to the AWS guide https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html